.github/workflows/assign-pr-author.yml

```diff
@@ -0,0 +1,12 @@
+# This GitHub action will auto-assign the PR author as an assignee and will not override the assignee if it's already assigned manually.
+
+name: Assign Author
+on: [pull_request]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: technote-space/assign-author@v1.3.1
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
can we just add one empty line in the end?

Reply:
I remember doing so, I think it was removed by GitHub file editor :thinking:

Reply:
I think maybe you have the code reformatting option being on before committing? 👀

Reply:
No it wasn't the IDE.. I created it using GitHub we interface